### PR TITLE
Make various minor changes

### PR DESCRIPTION
**Pull request by teekaytai**

### CODE REVIEW

**Review by eesung00 — COMMENTED** [review]
LGTM 👍

File: src/test/java/seedu/application/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -49,8 +49,8 @@ public class CommandTestUtil {
     public static final String POSITION_DESC_GOOGLE = " " + PREFIX_POSITION + VALID_POSITION_GOOGLE;
     public static final String POSITION_DESC_FACEBOOK = " " + PREFIX_POSITION + VALID_POSITION_FACEBOOK;
 
-    public static final String INVALID_COMPANY_DESC = " " + PREFIX_COMPANY + "James&"; // '&' not allowed in names
-    public static final String INVALID_CONTACT_DESC = " " + PREFIX_CONTACT + "911a"; // 'a' not allowed in phones
+    public static final String INVALID_COMPANY_DESC = " " + PREFIX_COMPANY + "James&"; // '&' not allowed in companies
```
Comment (jolynloh) [inline src/test/java/seedu/application/logic/commands/CommandTestUtil.java]: definitely nitpicking but maybe change to `Google&` to match application book? so other groups don't get to nitpick on our code when they review it :')

File: src/test/java/seedu/application/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -49,8 +49,8 @@ public class CommandTestUtil {
...
+    public static final String INVALID_CONTACT_DESC = " " + PREFIX_CONTACT + "911a"; // 'a' not allowed in contacts
     public static final String INVALID_DATE_DESC = " " + PREFIX_DATE + "01/09/2022"; // invalid date format
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
```
Comment (jolynloh) [inline src/test/java/seedu/application/logic/commands/CommandTestUtil.java]: same comment as above for email

**Review by jolynloh — COMMENTED** [review]
LGTM!

File: src/test/java/seedu/application/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (teekaytai) [reply src/test/java/seedu/application/logic/commands/CommandTestUtil.java]: Good idea

**Review by jhchee18 — COMMENTED** [review]
LGTM! 👍

**Pull request merged**